docker/docker-compose.yml

```diff
@@ -18,7 +18,7 @@ services:
       - ./config:/go/src/github.com/Disconnect24/Mail-GO/config
     ports:
       # Container 80 -> Host 8080
-      - "8080:80"
+      - "8080:8080"
```

Comment:
The default example config binds to container `:80` as expected: https://github.com/Disconnect24/Mail-GO/blob/364128942d783cbc5928aac7bc7cde7ca027ed9c/docker/config/config.json.example#L8

This shouldn't be done.

Reply:
I see this was resolved by your ca3b67cc1b796fd7c6a83b5f0564b4ff85906ab1, my bad.